api/index.js

```diff
@@ -401,7 +411,19 @@ export const inspectRequest = async (req, res) => {
   })
 }
 
-export const ingestEligibleDeals = async (req, res, client) => {
+/**
+ * @param {IncomingMessage} req
+ * @param {ServerResponse} res
+ * @param {pg.Client} client
+ * @param {string} dealIngestionAccessToken
+ */
+export const ingestEligibleDeals = async (req, res, client, dealIngestionAccessToken) => {
+  if (req.headers.authorization !== `Bearer ${dealIngestionAccessToken}`) {
+    res.statusCode = 403
+    res.end('Unauthorized')
+    return
```

Comment:
could be refactored with `status(res, 403)` from `http-responders`, but since we'll switch this to fastify soon, there is little motivation